web_poet/utils.py

```diff
@@ -15,3 +16,22 @@ def new_method(self, *args, **kwargs):
         return cache[self]
 
     return new_method
+
+
+def as_list(value: Optional[Any]) -> List[Any]:
+    """Normalizes the value input as a list.
+
+    >>> as_list(None)
+    []
+    >>> as_list("foo")
+    ['foo']
+    >>> as_list(123)
+    [123]
+    >>> as_list(["foo", "bar", 123])
+    ['foo', 'bar', 123]
+    """
+    if value is None:
+        return []
+    if not isinstance(value, list):
```

Comment:
It'd be nice to add a test for tuple, e.g. what should `as_list(("foo", "bar", 123))` return? Should it be a single-item list, or a tuple converted to a list?

Reply:
Good point! After looking into this further, I've added more input types that `as_list()` should also be able to handle and convert. Updated in https://github.com/scrapinghub/web-poet/pull/27/commits/5eab374d5767fe2fed821351a5c10fd1e9c9631e.